hm-service-account-api-rest-v1.yml

```diff
@@ -294,6 +294,43 @@ paths:
           description: A Service Account Token associated to a fleet application
       operationId: ApiWeb.FleetImportController.index
       description: Get the status of VINs that have previously been registered for data access clearance.
+  /v1/fleets/{vin}/cancel_activation:
+    post:
+      security:
+        - ServiceAccountToken: []
+      tags:
+        - Fleet
+      responses:
+        '200':
+          schema:
+            $ref: '#/definitions/FleetVin'
+          description: Vehicle will be canceled soon
+        '422':
+          schema:
+            $ref: '#/definitions/FleetErrors'
+          description: Invalid Input. It occurs when the payload is missing a params or the the application is not configured.
```

Comment:
Small typo with double "the" in the text. Can use:

"Invalid input. This occurs when the payload is missing a parameter or the application has not been configured."

Reply:
Done